include/c3d/uniforms.h

```diff
@@ -12,6 +16,13 @@ extern bool C3D_FVUnifDirty[2][C3D_FVUNIF_COUNT];
 extern bool C3D_IVUnifDirty[2][C3D_IVUNIF_COUNT];
 extern bool C3D_BoolUnifsDirty[2];
 
+/**
+ * @brief Provides a writable pointer for a floating-point uniform.
+ * @param[in] type \ref GPU_SHADER_TYPE of the uniform to be set.
+ * @param[in] id   ID of the uniform (retrivable using ctrulib's shaderInstanceGetUniformLocation())
+ * @param[in] size Amount of uniform registers allocated with this pointer.
```

Comment:
Do we want to use 'amount' here? Something like 'count' or 'number' sounds more natural

Reply:
Not sure if I am a fan of this language in general (it seems a bit confusing even if it is correct).
That being said, changing it to 'Number' would indeed make it sound more natural.